ceph_deploy/hosts/debian/__init__.py

```diff
@@ -16,6 +16,8 @@ def choose_init(module):
 
     Returns the name of a init system (upstart, sysvinit ...).
     """
+    if module.conn.remote_module.path_exists("/lib/systemd/system/ceph.target"):
+        return 'systemd'
```

Comment:
actually, no, this isn't right either... since we install the systemd unit file regardless.  we need to detect what the host is running...  ceph-detect-init?